tests/tt_metal/tt_metal/unit_tests_common/gtest_smoke/test_basic_pipeline.cpp

```diff
@@ -0,0 +1,351 @@
+                core,
+                {src_address, (uint32_t)src_noc_xy.x, (uint32_t)src_noc_xy.y, (uint32_t)num_tiles, (uint32_t)num_repetitions});
+        } else {
+            SetRuntimeArgs(
+                program,
+                receiver_kernels.at(core_id),
+                core,
+                {(uint32_t)device->worker_core_from_logical_core(cores[core_id - 1]).x,
+                 (uint32_t)device->worker_core_from_logical_core(cores[core_id - 1]).y,
+                 (uint32_t)num_tiles,
+                 (uint32_t)sender_semaphore_addr,
+                 (uint32_t)receiver_semaphore_addr,
+                 (uint32_t)num_repetitions});
+        }
+
+        if (core_id == num_cores - 1) {
+            SetRuntimeArgs(
+                program,
+                sender_kernels.at(core_id),
+                core,
+                {dst_address, (uint32_t)dst_noc_xy.x, (uint32_t)dst_noc_xy.y, (uint32_t)num_tiles, (uint32_t)num_repetitions});
+        } else {
+            SetRuntimeArgs(
+                program,
+                sender_kernels.at(core_id),
+                core,
+                {(uint32_t)device->worker_core_from_logical_core(cores[core_id + 1]).x,
+                 (uint32_t)device->worker_core_from_logical_core(cores[core_id + 1]).y,
+                 (uint32_t)num_tiles,
+                 (uint32_t)sender_semaphore_addr,
+                 (uint32_t)receiver_semaphore_addr,
+                 (uint32_t)l1_valid_value_addr,
+                 (uint32_t)num_repetitions});
+        }
+    }
+
+    ////////////////////////////////////////////////////////////////////////////
+    //                      Execute Application
+    ////////////////////////////////////////////////////////////////////////////
+    // send input data to the device
+    std::vector<uint32_t> src_vec =
+        create_random_vector_of_bfloat16(buffer_size, 100, std::chrono::system_clock::now().time_since_epoch().count());
+
+
+    log_info(LogTest, "Writing to device buffer...");
+    tt_metal::detail::WriteToBuffer(src_buffer, src_vec);
+    // fixture->WriteBuffer(device, src_buffer, src_vec);
+    log_info(LogTest, "Writing to device buffer Done.");
+
+    // EnqueueProgram(cq, program, false);
+    // Finish(cq);
+    fixture->RunProgram(device, program);
+
+    log_info(LogTest, "Kernels done.");
+
+    log_info(LogTest, "Reading results from device...");
+    std::vector<uint32_t> result_vec;
+    tt_metal::detail::ReadFromBuffer(dst_buffer, result_vec);
+    // fixture->ReadBuffer(device, dst_buffer, result_vec);
+
+    ////////////////////////////////////////////////////////////////////////////
+    //                      Validation & Teardown
+    ////////////////////////////////////////////////////////////////////////////
+    ASSERT_TRUE(src_vec == result_vec);
+}
+
+}  // namespace unit_tests::create_pipeline
+
+/*
+THIS TEST ALREADY EXISTS UNDER UNIT_TEST_FAST_DIPATCH/PIPELINING, REMEMBER TO REMOVE
```

Comment:
??

Reply:
test suites have been reorganized and this comment was removed/no longer applies